package.json

```diff
@@ -106,6 +106,7 @@
     "express": "^4.16.4",
     "lodash.throttle": "^4.1.1",
     "marinate": "^0.1.5",
+    "moment": "^2.24.0",
```

Comment:
This is [very heavy](https://bundlephobia.com/result?p=moment@2.24.0) library. See if you can use DOM APIs or date-fns.